docs/psa/accel/index.html

```diff
@@ -0,0 +1,4 @@
+<h1 id="psa-cryptographic-accelerator-interface">PSA cryptographic accelerator interface</h1>
```

Comment:
HTML in commit "Check in HTML versions" will need regeneration due to the `.md` updates.